featurebyte/routes/deployment/api.py

```diff
@@ -72,13 +72,16 @@ async def get_deployment(request: Request, deployment_id: PydanticObjectId) -> D
 
 @router.patch("/{deployment_id}")
 async def update_deployment(
-    request: Request, deployment_id: PydanticObjectId, data: DeploymentUpdate
+    request: Request, deployment_id: PydanticObjectId, data: DeploymentUpdate, response: Response
 ) -> Optional[Task]:
     """
     Update Deployment
     """
     controller = request.state.app_container.deployment_controller
     task: Task = await controller.update_deployment(document_id=deployment_id, data=data)
+    if isinstance(task, Task):
```

Comment:
Is the `task: Task` type annotation above accurate? It seems to return either a Task or something else, otherwise it seems like the condition here would always evaluate to True.